_projects/2018/astropy/astropy-learn-website.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Astropy Learn Website
+desc: Design and Implement landing page for Astropy Learn ecosystem.
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Web development
+ - User interface design and user experience
+difficulty: moderate
+issues:
+# Related issues (if any)  to this project.
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Kelle Cruz
+ - Lia Corrales
+ - Adrian Price-Whelan
+ - Erik Tollerud
+initiatives:
+ - GSOC
+ - SOCIS
+tags:
+# Different technologies needed
+ - python
+ - GUI
+ - web stuff
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - astropy
+---
+Design and Implement landing page for Astropy Learn ecosystem.
+
+#### Description
+create webpage front end for the Astropy Learn ecosystem which includes docs, examples, tutorials, and guides. Create a way to index and search for content. The implementation needs to be scalable.
+
+#### Milestones (if any)
+
+##### GSOC 2017 CODING STARTS
```

Comment:
Don't forget to update the year, we didn't update the template and a few people has failed to notice it :)